code/controllers/subsystem/processing/instruments.dm

```diff
@@ -0,0 +1,86 @@
+PROCESSING_SUBSYSTEM_DEF(instruments)
+	name = "Instruments"
+	init_order = INIT_ORDER_INSTRUMENTS
+	wait = 0.5
+	flags = SS_KEEP_TIMING
+	offline_implications = "Instruments will no longer play. No immediate action is needed."
+	stat_tag = "Instruments"
```

Comment:
This isnt needed, we can just have `P: x` on the statpanel for instruments, since everything else follows this format.